data/json/monsters.json

```diff
@@ -5729,6 +5765,7 @@
     "armor_bash":0,
     "armor_cut":0,
     "vision_day":30,
+    "vision_night":10,
```

Comment:
OK, difference of opinion then.  I'd wonder what made the woods more dangerous (all the zeds are in town?), you'd wonder why I'd still consider them safe.  Fair enough, we'll see how it goes.